src/components/Charts/DonutChart/DonutChart.tsx

```diff
@@ -0,0 +1,34 @@
+import { DonutChart as MantineDonutChart } from '@mantine/charts';
+import { Box, Group } from '@mantine/core';
+import { Typography } from '@/components/UI/Typography/Typography';
+import { CHART_COLORS } from '@/theme';
+import { DonutChartPropTypes } from '../chart.types';
+import { chartStyles } from './styles';
+
+export default function DountChart({ data }: DonutChartPropTypes) {

```

Comment:
Can you change it into `DonutProps` ? 
It shouldnt be plural because its not referring to multiple types.

And specifically put the props interface inside the same file as the component